api/proto/teleport/legacy/types/types.proto

```diff
@@ -6595,6 +6595,33 @@ message AWSICProvisioningSpec {
   string bearer_token = 2;
 }
 
+// PluginAWSICStatusV1 defines AWS Identity Center plugin sub-process status.
+message PluginAWSICStatusV1 {
+  // GroupImportStatus is a status of Identity Center group and group members import.
+  AWSICGroupImportStatus group_import_status = 1;
+}
+
+// AWSICGroupImportStatus defines Identity Center group and group members import status.
+message AWSICGroupImportStatus {
+  // StatusCode is a status code of group and group members import operation.
+  AWSICGroupImportStatusCode status_code = 1;
+  // ErrorMessage contains error message for a group and group members import attempt
+  // that met with an error.
+  string error_message = 2;
+}
+
+// AWSICGroupImportStatus defines Identity Center group and group members
+// import status codes.
+enum AWSICGroupImportStatusCode {
+  // UNSPECIFIED denotes that a status is unknown.
+  UNSPECIFIED = 0;
+  // DONE denotes that the group and group members import operation was
+  // performed with success.
+  DONE = 1;
```

Comment:
Nit: Maybe "COMPLETED" or "SUCCESS" (to match "ERROR")? Also, does this really need to be an enum? Would just having a string simplify things?

Reply:
Updated error to be "Failed" but kept "Done". 
Using enum because:
- Since we already need to define proto type for the status field, enum makes it easy to define values in the proto itself. Otherwise, I will still need to maintain another set of constants to ensure correct string values are supplied.
- It makes it easy if we want to extend the possible state, say add a pending state etc.